tests/extra_function_tests.rs

```diff
@@ -469,7 +469,7 @@ fn ltime_to_lword_conversion() {
     VAR
         t: TIME := TIME#12h1m20s391ms10ns;
     END_VAR
-        main := TOD_TO_LWORD(t);
+        main := TIME_TO_LWORD(t);
```

Comment:
the test is called ltime_to_lword...
we have TIME and are calling TIME_TO_LWORD
maybe rename the test to time_... and add another one for ltime
haven't reviewed all existing tests but maybe there are other tests with the same issue